### ISSUE TITLE
SN01 not returning GPS data to GroundStation

### ISSUE THREAD

**Issue by nickcounts** [xinabox/xSN01#2]
I (and several of our users) have been unable to get valid GPS data from the SN01 since 1.316 (maybe since 1.315, not totally sure). All my testing returns data that looks like this:

```
"SN01": {
      "HDOP": 10,
      "Number Satellites": 0,
      "Status": "OFFLINE"
    },
```

I don't see anything that looks like a major change in the SN01 library, so I'm at a bit of a loss. Any ideas for us?

CR03 Firmware V1.318 (CR03_Firmware_V1_318_ID01_ID02.hex)
CW01 Firmware V1.316

**Comment by brendanvanbreda** [comment]
@nickcounts we have found the above mentioned issue during out testing for CR03 Firmware. An update will be pushed to to xSN01.

**Comment by brendanvanbreda** [comment]
Fixed in 4128782283f1e72efa8a292c6ccf5a40a3e19ddd.
